examples/wistia/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "wistia",
+  "version": "1.0.0",
+  "description": "Vistia  Widget for contentful",
+  "main": "./src/app.js",
+  "scripts": {
+    "env": "env",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "widget:create": "./node_modules/.bin/better-npm-run widget:create",
+    "widget:update": "./node_modules/.bin/better-npm-run widget:update",
+    "widget:dev": "./node_modules/.bin/better-npm-run widget:dev",
+    "widget:serve": "./node_modules/.bin/better-npm-run widget:serve"
```

Comment:
No need to prefix these with `./node_modules/.bin`. That folder is already included in the path